presto-docs/src/main/sphinx/release/release-0.272.rst

```diff
@@ -0,0 +1,40 @@
+=============
+Release 0.272
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix parquet data page offset calculation in parquet writer.
+* Ensure Parameters are in proper order for queries having WITH clause.
+* Enable cost based optimization by default.
+* Add a new JDBC driver parameter - timeZoneID :doc:`/installation/jdbc` :issue:`16680`.
+* Add secure_random() and secure_random(lower, upper) with uniform distribution.
+* Add support to use Parquet page-level statistics to skip pages.
+
+Mongodb Connector Changes
+_________________________
+* Fix the spelling of the write concern option ``JOURNAL_SAFE`` for the property ``mongodb.write-concern``.
+
+Hive Changes
+____________
+* Introduce a flag "hive.parquet-column-index-filter-enabled" to turn on/off this feature. By default it is off.
```

Comment:
Add support for using Parquet page-level statistics to skip pages. This feature can be enabled by setting the ``hive.parquet-column-index-filter-enabled`` configuration property.